admin.php

```diff
@@ -46,9 +46,22 @@
 <script type="text/javascript" src="js/plugins/jquery.js"></script>
 <script type="text/javascript" src="js/plugins/thickbox.js"></script>
 <script type="text/javascript" src="js/common.js"></script>
+<script type="text/javascript" src="js/plugins/Gettext.js"></script>
+<?php
+   if(isset($_COOKIE["lang"])){
+        if($_COOKIE["lang"]=='fr'){

```

Comment:
This code works for French/English, but it will be hard to maintain as we add more languages. We should have an array of language codes so we can say "href='./locale/" . $lang_codes[$_COOKIE["lang"]] . "/LC_MESSAGES/messages.po'". The translation wiki page says you plan to do something like this, is that correct?